app/Http/Controllers/Admin/ContentController.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Requests;
+use Illuminate\Database\Eloquent\Model;
+use App\Http\Controllers\Controller;
+use App\Models\Content;
+use App\Models\Category;
+
+class ContentController extends Controller {
+
+    private $contentModel;
+    private $categoryModel;
+
+    public function __construct(Content $contentMode, Category $categoryModel) {
+        parent::__construct();
+
+        $this->contentModel = $contentMode;
+        $this->categoryModel = $categoryModel;
+    }
+
+    public function index() {
+        $contents = $this->contentModel->take(20)
+                ->orderBy('updated_at', 'desc')
+                ->get();
+
+        return view('admin/content/index', [
+            'contents' => $contents
+        ]);
+    }
+
+    public function getCreate() {
+        $categories = $this->categoryModel
+                ->orderBy('updated_at', 'desc')
+                ->get();
```

Comment:
Indent đang sai anh ạ. 1 là cái orderBy và get chỉ vào 1 tab so với $content =. Hoặc anh đẩy như thế này (bọn em thường làm như vậy):

$contents = $this->contentModel->take(20)
                               ->orderBy('updated_at', 'desc')
                               ->get();